http-auth/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"http-auth/routes"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/joho/godotenv"
+	"golang.org/x/time/rate"
+)
+
+func exec() {
```

Comment:
What calls this? Is anything actually causing the `.env` file to be read?

Reply:
It was supposed to be `init()` function so it loads env before everything.